index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="Stylesheet" href="./CSS/style.css"/>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Cynthia's portfolio</title>
+  </head>
+
+    <body>
+      <header class="main-header d-flex">
+        <h2>Welcome</h2>
+        <nav class="nav d-flex">
+          <ul class="nav-menu d-flex flex-center">
+            <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+            <li class="nav-link active"><a href="#About">About</a></li>
+            <li class="nav-link"><a href="#Contact">Contact</a></li>
+          </ul>
+          <img src="./image/icons/Hamburger_icon.svg.png" class="show-menu-btn" alt="union">
+          <img src="./image/icons/menu-close.svg" class="closed-menu-btn" alt="">
+        </nav>
+      </header>
+      <main class="container">
+        <section class="main-section">
+          <div class="main-section-title">
+            <h1 class="h2-size">Hey there! I'm Cynthia Inga</h1>
+            <h2 class="h2-size">I'm a software developer <br/> and Engineer in Electronics</h2>
+          </div>
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don't hesitate to contact me.</p>
+          <div class="icons-container">
+            <ul class="social-media-icons d-flex flex-center">
+              <li>
+                <a href="https://twitter.com/CynthiaInga_C">
+                  <img src="./image/icons/twitter-icon.png" alt="twitter">
+                </a>
+              </li>
+              <li>
```

Comment:
[OPTIONAL}: Please ensure to align the content of this section with flexbox according to the requirement. It's marked as OPTIONAL because it was not mentioned by the previous reviewer.